Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneEventCollector/TrendMicroVisionOneEventCollector_test.py

```diff
@@ -31,14 +31,35 @@ def get_url_params(url: str) -> Dict[str, str]:
     }
 
 
-def create_any_type_logs(start: int, end: int, created_time_field: str, id_field_name: str, extra_seconds: int = 0):
+def create_any_type_logs(
+    start: int,
+    end: int,
+    created_time_field: str,
+    id_field_name: str,
+    last_event_fetch_time: str | None,
+    extra_seconds: int = 0,
+    ascending_order: bool = False,
+):
+    last_event_fetch_time = last_event_fetch_time or (datetime.now() - timedelta(minutes=1)).strftime(DATE_FORMAT)
+    if ascending_order:
```

Comment:
thought last_event_fetch_time is only relevant for descending?

Reply:
relevant for both, i just use the last run time to create events from this time forward, its only for testing purposes

Reply:
generally testing code shouldnt be in the production code

Reply:
didnt realize this was test file